string_match_without_scan_or_regex_spec.rb

```diff
@@ -0,0 +1,144 @@
+require_relative 'string_match_without_scan_or_regex'
+
+describe String do
+
+  let(:randomly_generated_strings){ (0..2).map{(0..rand(1..100)).map{(rand(127)).chr}.join}}
+  let(:string){ randomly_generated_strings[0] + "like" + randomly_generated_strings[1] + "like" + randomly_generated_strings[2]}
+  let(:substring){"like"}
+  let(:substring_two){"nope"}
+
+
+  #Testing to ensure above variables used in the spec are objects of the class String
+  describe 'string variables' do
+    describe 'string' do
+      it 'should be an object of the class String' do
+        expect(string).to be_kind_of(String)
```

Comment:
This test, and the ones below, are only testing your spec.  I'd get rid of them, there's no value here.